qa/rolling-upgrade/build.gradle

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
+
+apply from : "$rootDir/qa/build.gradle"
+
+String knn_bwc_version = System.getProperty("bwc.version")
+String baseName = "knnBwcCluster-rolling"
+
+// Creates a test cluster of previous version and loads k-NN plugin of bwcVersion
+testClusters {
+    "${baseName}" {
+        testDistribution = "ARCHIVE"
+        versions = [knn_bwc_version, opensearch_version]
+        numberOfNodes = 3
+        plugin(project.tasks.zipBwcPlugin.archiveFile)
+        setting 'path.repo', "${buildDir}/cluster/shared/repo/${baseName}"
+        setting 'http.content_type.required', 'true'
+        environment "LD_LIBRARY_PATH", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/knnlib"
+        systemProperty "java.library.path", "${buildDir}/testclusters/${baseName}-0/distro/${knn_bwc_version}-ARCHIVE/plugins/opensearch-knn/knnlib"
+    }
+}
+
+
+// Task to run BWC tests against the old cluster
+task "${baseName}#testAgainstOldCluster"(type: StandaloneRestIntegTestTask) {
```

Comment:
WHy use "${baseName}" variable here? I think that if we are only creating 1 cluster, its okay to just to name it testAgainstOldCluster? Same with below tasks.

Reply:
ack